supabase/migrations/20260902132624_expand_webnative_onboarding.sql

```
@@ -0,0 +1,62 @@
+ALTER TABLE "public"."orgs"
+DROP CONSTRAINT "orgs_onboarding_valid";
+
+ALTER TABLE "public"."orgs"
+ADD CONSTRAINT "orgs_onboarding_valid" CHECK (
+  ("jsonb_typeof"("onboarding") = 'object'::"text")
+  AND (
+    (NOT ("onboarding" ? 'intent'::"text"))
+    OR (
+      ("onboarding" ->> 'intent'::"text")
+      = ANY (ARRAY['unknown'::"text", 'ota'::"text", 'builder'::"text", 'both'::"text", 'exploring'::"text", 'publish'::"text"])
```

SQLFluff (4.3.0)

[error] Line is too long (128 > 80).

(LT05)